### ISSUE TITLE
React does not recognize the `maxSize` prop on a DOM element

### ISSUE THREAD

**Issue by dkadrios** [nathancahill/split#563]
After updating from 2.0.9 to 2.0.10 I've started encountering the following console error: 

```
Warning: React does not recognize the `maxSize` prop on a DOM element.  If you intentionally 
want it to appear in the DOM as a custom attribute, spell it as lowercase `maxsize` instead. If 
you accidentally passed it from a parent component, remove it from the DOM element.
    at div
    at construct (/app/node_modules/react-split/dist/react-split.js:17:24)
```

It looks like `maxSize` is now getting spread into the props of the component or it's children.  We're not specifying maxSize, only `className` and `gutterSize`.  I tried passing undefined, null, zero, etc for maxSize to try and negate it but the error persists.

**Comment by nathancahill** [comment]
Investigating, that is a new property that should not be passed down.

**Comment by nathancahill** [comment]
Fixed in 2.0.11.

**Comment by dkadrios** [comment]
Thanks for the fast response, @nathancahill!
